puppet-mode.el

```diff
@@ -275,12 +275,19 @@ Return nil, if there is no special context at POS, or one of
   "Launch a Puppet REPL using `puppet-repl-command' as an inferior mode."
   (interactive)
 
+  ;; the command (at least for now) is 'puppet debugger'
+  ;; but really it is 'puppet' with args 'debugger' + puppet-repl-args
+  ;; Even if it changes, we need to not break on multi-word commands
   (unless (comint-check-proc puppet-repl-buffer)
-    (set-buffer
-     (apply 'make-comint "Puppet-REPL"
-            puppet-repl-command
-            nil
-            puppet-repl-args))
+    (let* ((command-list (split-string puppet-repl-command))
+           (real-puppet-repl-command (car command-list))
+           (real-puppet-repl-args (append (cdr command-list)
```

Comment:
what's the chance of the puppet debugger command changing (again) and not being a `face` anymore?